src/praisonai/praisonai/sandbox/daytona.py

```
@@ -1,464 +1,55 @@
+    Requires: Daytona CLI and workspace access
     """
     
     def __init__(
         self,
-        workspace_template: str = "python",
-        provider: str = "local",
-        workspace_name: Optional[str] = None,
-        api_key: Optional[str] = None,
-        server_url: Optional[str] = None,
-        timeout: int = 300,
+        workspace_name: str = None,
```

Ruff (0.15.12)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)